bot/exts/moderation/watchchannels/talentpool.py

```diff
@@ -101,8 +98,12 @@ async def watch_command(self, ctx: Context, user: FetchedMember, *, reason: str
         async with session.post(url, **kwargs) as resp:
             response_data = await resp.json()
 
-            if resp.status == 400 and response_data.get('user', False):
-                await ctx.send(":x: The specified user can't be found in the database tables")
+            if resp.status == 400:
+                if response_data.get('user', False):
+                    await ctx.send(":x: The specified user can't be found in the database tables")
+                elif response_data.get('actor', False):
+                    await ctx.send(":x: You already have nominated this user")
```

Comment:
I think "You have already nominated this user" sounds better
```suggestion
                    await ctx.send(":x: You have already nominated this user")
```